ipfs-cluster/trigger-server/README.md

```diff
@@ -1,3 +1,9 @@
+# What is trigger server
+
+The trigger server's purpose is defined in ipfs/ipfs-cluster/issues/244. The goal is to create an endpoint
+which can be triggered remotely and automated to run the tests, obtaining their results.
+
+
```

Comment:
Explain that it runs in the kubernetes master? Explain the authentication mechanism and how to set it up? Explain how to put it there after you have the packages? explain that it includes a systemd unit file? how to obtain credentials or where to reach out if needed...